### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by matteo-prosperi**
New issue template for aws-tools-for-powershell

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report-or-feature-request.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+**Expected Behavior**
+If you're describing a bug, tell us what should happen. If you're suggesting a change/improvement, tell us how it should work.
+
+**Current Behavior**
+If describing a bug, tell us what happens instead of the expected behavior.
+Include full errors, uncaught exceptions, stack traces, and relevant logs.
+If service responses are relevant, please include Fiddler-captured wirelogs.
+If suggesting a change/improvement, explain the difference from current behavior.
+
+*Possible Solution***
```
Comment (sstevenkang) [inline .github/ISSUE_TEMPLATE/bug-report-or-feature-request.md]: Incorrect bolding.

Comment (matteo-prosperi) [reply .github/ISSUE_TEMPLATE/bug-report-or-feature-request.md]: Fixed

File: .github/ISSUE_TEMPLATE/bug-report-or-feature-request.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+**Expected Behavior**
+If you're describing a bug, tell us what should happen. If you're suggesting a change/improvement, tell us how it should work.
+
+**Current Behavior**
+If describing a bug, tell us what happens instead of the expected behavior.
+Include full errors, uncaught exceptions, stack traces, and relevant logs.
+If service responses are relevant, please include Fiddler-captured wirelogs.
+If suggesting a change/improvement, explain the difference from current behavior.
+
+**Possible Solution***
```
Comment (sstevenkang) [inline .github/ISSUE_TEMPLATE/bug-report-or-feature-request.md]: it looks like i missed it the first time.  this one has extra *

Comment (matteo-prosperi) [reply .github/ISSUE_TEMPLATE/bug-report-or-feature-request.md]: Done

**Comment by sstevenkang** [comment]
LGTM.

**Pull request merged**